lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -1691,12 +1690,11 @@ private function generateIdentifierName($identifier)
     protected function getCommentOnTableSQL(string $tableName, ?string $comment): string
     {
         return sprintf(
-            <<<'SQL'
+            "
```

Comment:
Why is this change necessary?

Reply:
HEREDOCs were added in PHP 7.3

Reply:
Correction, moving the closing tag for now/heredocs to the same indentation as starting was added in 7.3

Reply:
Won't moving the heredoc terminator to the beginning of the line have more or less the same effect?

Reply:
It applies a tab more for every line, but works.
imho it's better to not indent heredoc blocks.

```PHP
return sprintf(
            <<<SQL
EXEC sys.sp_addextendedproperty @name=N'MS_Description',
    @value=N%s, @level0type=N'SCHEMA', @level0name=N'dbo',
@level1type=N'TABLE', @level1name=N%s
SQL,
    $this->quoteStringLiteral((string) $comment),
    $this->quoteStringLiteral($tableName)
);
```

Reply:
@beberlei you could use `git revert 9a8ea5e039268368d48f38c853a00840e8201bde` to make it 7.1-compatible